Sage/carbon
# docs(flat-table): add placeholder to inputs in custom column width story FE-4405

fix #4490

### Proposed behaviour


Adds placeholder to inputs to ensure the demo is accessible and does not throw any axe warnings

Fixes potential accessibility issues where a `aria-labelledby` was being set on the
`HiddenCheckableInput` with a `labelId` that did not correspond to an existing `Label`. Surfaces an
`ariaLabelledBy` prop to support overriding when necessary.

Adds `ariaLebelledBy` prop to interface to allow implementation teams to pass id of labelling
element and improve accessibility for users

### Current behaviour


Accessibility warnings on all stories that use the `FlatTableCheckbox` component and the custom column width FlatTable story

aria-labelledby is set on `HiddenCheckableInput` when there is no associated label

No way of overriding `ariaLabelledBy`

### Checklist




#### QA


### Additional context



### Testing instructions


Check FlatTable (expandable ones as well) stories against master for accessibility warnings 


The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
